app/models/presenter.rb

```diff
@@ -0,0 +1,15 @@
+class Presenter
+
+  attr_reader :items, :categories,
+  def initialize
+    @items = Item.all
+    @categories = Category.all
+  end
+
+  def unique_category_names
+    Category.all.each do |category|
+      category.name
+    end.uniq
+    require 'pry' ; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.<br>Space found before semicolon.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.